src/components/title.tsx

```diff
@@ -40,6 +42,7 @@ const StyledTitle = styled(Title)`
 
 StyledTitle.defaultProps = {
   titleSize: "medium",
+  color: "currentcolor",
```

Comment:
the default `color` should be assigned to a valid css value. Like `black` or `#000`.

Reply:
It does seem like `currentcolor` is a valid css value, but at the same time, this is not how it's supposed to be used, according to [CSS Tricks](https://css-tricks.com/currentcolor/).

> you can use this value to indicate you want to use the value of color for other properties that accept a color value: borders, box shadows, outlines, or backgrounds.

How about `color: inherit;`? I'm a bit hesitant to assign a default color because some times I want to use a wrapper with a color for the entire box rather than setting a color to each element:

```jsx
<Text align="center" color={colors.graySemibold}>
  <Title>
    Contact Us
  </Title>

  <Text>
    Have questions? Get in touch:&nbsp;
    <TextLink href="#">youremail@example.com</TextLink>
  </Text>
</Text>
```

Reply:
Ah ok! I stand corrected then, never seen it before 😅. 
Right, `color: inherit` would give us the behavior you described. Let's go with that